applications/source/rabbit-source/README.adoc

```diff
@@ -0,0 +1,103 @@
+//tag::ref-doc[]
+= RabbitMQ Source
+
+The "rabbit" source enables receiving messages from RabbitMQ.
+
+The queue(s) must exist before the stream is deployed; they are not created automatically.
+You can easily create a Queue using the RabbitMQ web UI.
+
+== Input
+
+N/A
+
+== Output
+
+=== Headers
+
+* `content-type: text/plain`
+
+=== Payload
+
+* `String`
+
+=== Headers
+
+* `content-type: application/octet-stream`
+
+=== Payload
+
+* `byte[]`
+
+=== Headers
+
+* `content-type: application/x-java-serialized-object`
+
+=== Payload
+
+* `java.io.Serializable`
+
+== Options
+
+The **$$rabbit$$** $$source$$ has the following options:
+
+//tag::configuration-properties[]
+$$rabbit.enable-retry$$:: $$true to enable retry.$$ *($$Boolean$$, default: `$$false$$`)*
```

Comment:
Looks like some of these properties don't reflect reality.
Your new `RabbitSupplierProperties` has a `rabbit.supplier` prefix...

I think there is a docs plugin which can regenerate all of these properties according the current `dataflow-configuration-metadata-whitelist.properties` configuration.

@sobychacko , any advises why this is not an actual?

Thanks

Reply:
@artembilan yes, I took the old aReadme.doc without changes. I'll update it.